src/IssueViz.Security/Hotspots/HotspotsStore.cs

```diff
@@ -65,14 +59,22 @@ public IAnalysisIssueVisualization GetOrAdd(IAnalysisIssueVisualization hotspotV
                 return existingHotspot;
             }
 
+            var oldIssues = hotspots.ToArray();
+
             hotspots.Add(hotspotViz);
 
+            NotifyIssuesChanged(oldIssues);
```

Comment:
The only change to the collection is the addition of a new issue and it isn't part of the notification. Is this correct?

Reply:
Ah - I see. Seems a bit weird to return all of the old and new issue in the event arguments. I missed that before - I incorrectly interpreted it as "removed" and "added".

Reply:
Ah, but I do realize that there is a bug in the StoreAdapter: I need to notify only on what's changed, so a diff between old and new 🤦 I'll fix that in a separate PR.